Remove empty country names from es-MX.yml (#160)
solidusio/solidus_i18n: README.md

gem 'solidus_globalize', github: 'solidusio-contrib/solidus_globalize', branch: 'master'
```

## Localizing country names

> Feel free to define `spree.country_names` in your own locale files if you need custom localization of country names. The value expected is a hash. For example, to have countries in Spanish do:

Reviewer: ```suggestion
You can translate country names by defining `spree.country_names` in your own locale files. For example, to have countries in Spanish do:
```

```yml
es:
  spree:
    country_names:

>       # ISO: Country name

Reviewer: ```suggestion
```
